src/nl/hannahsten/texifyidea/util/Magic.kt

```diff
@@ -184,7 +184,8 @@ object Magic {
                 "table", "tab",
                 "tabular", "tab",
```

Comment:
It was already here, but I think I'll remove it as tabular isn't floating, quote from https://www.tug.org/TUGboat/tb28-3/tb90hoeppner.pdf :
>  To typeset  material  in  rows  and  columns, tabular is needed, while the table environment is a container for floating material similar to figure, into which a tabular environment may be included.